source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst

```diff
@@ -10,8 +10,8 @@ Initial DNS Seedlist Discovery
 :Authors: Derick Rethans
 :Status: Draft
 :Type: Standards
-:Last Modified: 2019-04-15
-:Version: 1.4.0
+:Last Modified: 2021-09-xx
+:Version: 1.4.1
```

Comment:
I think this warrants a minor version bump, as was done for the URI options spec -- but I won't push for that if other feel strongly about the SRV and Polling SRV specs keeping patch bumps.

Reply:
I agree here - a new minor version would also indicate to teams that there may be some effort required to support the new spec version, as opposed to only syncing spec tests and hoping for the best ;)

Reply:
I would argue that this is more a `1.4.0` -> `1.5.0`. When I see `1.4.0` -> `1.4.1`, I'm expecting spelling corrections and clarifications, not new functionality. I'm not going to insist, but just my 2 cents.

Reply:
Apologies. I was looking at an outdated diff. You did bump to `1.5.0`. I'm good with that.